.travis.yml

```diff
@@ -9,20 +12,25 @@ cache:
     - admin/node_modules
     - client/node_modules
 
+env:
+  global:
+    - APP_ENV=prod
+
+# - if [ "TRAVIS_PULL_REQUEST" != "false" ]; then exit 0; fi
+before_install:
+  - curl -O https://storage.googleapis.com/kubernetes-helm/helm-v2.7.2-linux-amd64.tar.gz
```

Comment:
We can use `curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get | bash -s -- --version v2.9.1` instead